examples/Getting started with the Amazon S3 Connector for PyTorch.ipynb

```diff
@@ -643,7 +643,7 @@
     {
      "data": {
       "text/plain": [
-       "'2022.001.0000.00.DISC.01.jpg.image.jpg'"
+       "['2022.001.0000.00.DISC.01.jpg.image.jpg']"
```

Comment:
I am probably missing something, but why do we need this change?

Reply:
Using `DataLoader` instead of a plain iterator means we're using PyTorch's batch system. 
This means that our data will always be batched, so the behaviour on iteration changes. The example here uses size 1, so to get the data we want out, you have to do indexing.